terraform/files/bin/add_cluster-network.sh

```diff
@@ -25,7 +25,8 @@ findnewnic()
 MGMT="$PREFIX-mgmtcluster"
 # FIXME: We already know the name ($PREFIX-mgmtcluster)
 #MGMT=$(openstack server list --name "$PREFIX-mgmtcluster" -f value -c Name)
-openstack server add network $MGMT k8s-clusterapi-cluster-default-$CLUSTER_NAME || exit
+NETWORK=$(openstack network list -f value -c Name | grep "k8s-clusterapi-cluster-\(default-${CLUSTER_NAME}\|${CLUSTER_NAME}-${CLUSTER_NAME}\)")
+openstack server add network $MGMT $NETWORK || exit
```

Comment:
Wouldn't just
`openstack server add network $MGMT k8s-clusterapi-cluster-${CLUSTER_NAME}-${CLUSTER_NAME}` do the job?
Or are you concerned about an old management host with old clusters (that were created before we used namespaces) and then the mgmt host got updated?